server/channels/app/opentracing/opentracing_layer.go

```diff
@@ -18,14 +18,16 @@ import (
 	"reflect"
 	"time"
 
+	"github.com/opentracing/opentracing-go/ext"
```

Comment:
This seems out of place.

Reply:
This is a result of adding the `github.com/mattermost/mattermost/server/public/shared/request` import, even if I put in above this one it'll end up there. I [dont see any options](https://cs.opensource.google/go/x/tools/%20/refs/tags/v0.12.0:imports/forward.go;l=18) to modify that behaviour.

Reply:
Are you sure you are using Go 1.19.x? This seems like the formatting behavior in a more recent version.

Reply:
> Are you sure you are using Go 1.19.x? This seems like the formatting behavior in a more recent version.

I have it installed from brew so I most likely have 1.21 installed. Let me fall back to 1.19.

Reply:
> Are you sure you are using Go 1.19.x? This seems like the formatting behavior in a more recent version.

I've just downgraded to 1.19, cleaned up everything, rebuilt the `struct2interface` binary and I'm getting the same results.